docs/interactions.rst

```diff
@@ -0,0 +1,126 @@
+:orphan:
+
+.. currentmodule:: nextcord
+
+.. _interactions:
+   
+Slash Commands
+==============
+
+Since Discord has added interaction commands a feature with many possibilities, we at Nextcord have decided to add them to our fleet of features.
+
+This doc will explain the innerworkings and how to use interaction commmands.
+
+We suggest you learn how to make regular commands before looking through here, we suggest looking at the :doc:`quickstart`
+
+How To Make A Simple Slash Command
+----------------------------------
+
+This right here is a simple ping pong command made with Nextcord's slash feature.
+
+.. code-block:: python3
+
+    @client.slash_command()
+    async def ping(interaction):
+        await interaction.response.send_message("Pong!")
+        
+The way it works is that you use the :meth:`~Client.slash_command` function to interact with discord's application commands endpoint.
```

Comment:
```suggestion
The way it works is by using:meth:`~Client.slash_command` decorator to add a slash command for the bot to register with Discord.
```

Reply:
@vcokltfre shouldnt it be *the* @slash_command deco...